src/precompiles/mod.rs

```diff
@@ -87,7 +118,8 @@ pub fn homestead_precompiles(
         #[cfg(feature = "exit-precompiles")]
         ExitToEthereum::ADDRESS => Some(ExitToEthereum::run(input, target_gas, context)),
         _ => None,
-    }
+    };
+    output.map(|res| res.map(|o| o.into()))
```

Comment:
Could avoid the closure and write `output.map(|res| res.map(Into::into))`. Probably doesn't matter much either way, but maybe slightly easier to read without the closure.

(same applies to Byzantium, Istanbul, etc.)

Reply:
Ah thats right, I forgot you could do that now. I'm indifferent either way.

Reply:
I like the version without closure slightly better, so changed in https://github.com/aurora-is-near/aurora-engine/pull/116/commits/613982ca942cacb2784dcfb09209b2c55b513b8c